server/routes.go

```diff
@@ -1572,8 +1572,12 @@ func (s *Server) ChatHandler(c *gin.Context) {
 				sb.Reset()
 				ch <- res
 				return
+			} else {
+				if !strings.HasPrefix(sb.String(), "{") {
+					ch <- res
+					return
+				}
```

Comment:
```suggestion
			} else if !strings.HasPrefix(sb.String(), "{") {
					ch <- res
					return
			}
```

Reply:
actually not going with this pattern - this won't work for models like mistral so going to hack on the templates instead